wikichange/src/content/content.js

```diff
@@ -210,26 +212,26 @@ const renderItemsBelowGraph = async (creationDate) => {
     let curRevisionId = revisions[0][0];
     const oldRevision = revisions[1];
     let oldRevisionId = oldRevision[0];
-    const oldRevisionDate = oldRevision[1].toLocaleDateString().slice(0, 10);
-    belowGraphDiv.innerHTML = `<div id="loader"></div><button class="extensionButton" id="highlightButton">Highlight Changes From</button>
-    <input type="date" value="${initialDate
-        .toISOString()
-        .slice(0, 10)}" id="dateOutput" name="dateOutput" style="text-align: center;"> 
-                                <p></p>
-                                <button class="extensionButton" id="revisionButton">${getTextForRevisionButton(
-                                    oldRevisionDate
-                                )}</button>
-                            <div style="padding-left: 3%; padding-top: 3%; text-align: center;">
-                                <div class="card" style="border-style: solid;">
-                                    <div class="card-body" style="text-align: center;">
-                                    <p class="card-text" id="revisionDate"> ${getRevisionToClosestDateText(
-                                        getRevisionPageLink(title, curRevisionId, oldRevisionId).replace(/\s/g, "_"),
-                                        oldRevisionDate
-                                    )}</p>
-                                    <p class="card-text"> Newly added texts are highlighted in green, but the deletions are not included </p>
-                                    </div>
-                                </div>
-                            </div>`;
+    const oldRevisionDate = oldRevision[1].toLocaleDateString("en-US").slice(0, 10);
+    highlightRevisionBetweenRevisionIds(title, curRevisionId, oldRevisionId);
```

Comment:
Why are we calling `highlightRevisionBetweenRevisionIds` here?

Reply:
I don't think this is my code

Reply:
It's added in your pr, so u can remove it then.